qa/standalone/ceph-helpers.sh

```diff
@@ -560,6 +560,7 @@ function run_mgr() {
         --admin-socket=$(get_asok_path) \
         --run-dir=$dir \
         --pid-file=$dir/\$name.pid \
+        --mgr-module-path=$(realpath ${CEPH_ROOT}/src/pybind/mgr) \
```

Comment:
I already have this change in my branch, but I'll testing with the run-standalone.sh change.